spec/models/scenarios_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Scenario, type: :model do
+  context 'instance attributes and methods' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.